src/pages/index.vue

```diff
@@ -943,18 +1012,24 @@ export default {
     bookstoreSectionStickyClass() {
       return 'desktop:sticky desktop:top-[124px]';
     },
-    bookstoreItems() {
+    editorialBookstoreItems() {
       return this.nftBookstoreCMSProductsForLandingPage.filter(
         item =>
           !item.locales ||
           item.locales.some(locale => this.$i18n.locale.includes(locale))
       );
     },
-    stickyBookstoreItem() {
-      return this.bookstoreItems[0];
+    stickyEditorialBookstoreItem() {
+      return this.editorialBookstoreItems[0];
+    },
+    editorialBookstoreItemsInGrid() {
+      return this.editorialBookstoreItems.slice(1, 10);
     },
-    bookstoreItemsInGrid() {
-      return this.bookstoreItems.slice(1, 10);
+    latestBookstoreItems() {
+      return this.nftGetBookstoreCMSProductsByTagId('landing-latest').slice(
+        0,
+        10
+      );
```

Comment:
Should remove item exists in `editorialBookstoreItems`?

Reply:
handled in Airtable